docs/upgrade/upgrade-security.asciidoc

```diff
@@ -13,35 +13,23 @@ NOTE: For detailed information about the 8.0 release, check out the <<release-no
 [[upgrade-reqs]]
 == Upgrade requirements
 
-*Track rules that are automatically disabled when upgrading*
-
-Upon upgrading to 8.0, rules are automatically disabled. Once the upgrade completes, you should re-enable them again to avoid gaps in rule coverage. _We highly recommend that you track your active rules before upgrading so you can easily find and re-enable them after upgrading to 8.0_.
-
-Before upgrading, use the <<rules-api-find, Find rules API>> to capture a list of enabled rules.
+*Upgrade {agent}s*
 
-IMPORTANT: Use curl or another HTTP tool to run {elastic-sec} API commands.
+Upgrade your {stack} and {agent}s to 7.17 first (refer to {fleet-guide}/upgrade-elastic-agent.html[Upgrade Fleet-managed Elastic Agents]). After this is complete, you can  {stack-ref}/upgrading-elastic-stack.html[upgrade the {stack}] to 8.0. Initially, {agent}s will be version 7.17; this is fine because {elastic-sec} supports the last minor release in 7.x (7.17) and any subsequent {elastic-endpoint} versions in 8.x. After the {stack} upgrade, you can decide whether to upgrade {agent}s to 8.0, which is recommended to ensure you get the latest features.
```

Comment:
```suggestion
Upgrade your {stack} and {agent}s to 7.17 first (refer to {fleet-guide}/upgrade-elastic-agent.html[Upgrade Fleet-managed Elastic Agents]). Afterwards, you can  {stack-ref}/upgrading-elastic-stack.html[upgrade the {stack}] to 8.0. Initially, {agent}s will be version 7.17; this is fine because {elastic-sec} 8.x supports the last minor release in 7.x (7.17) and any subsequent {elastic-endpoint} versions in 8.x. After the {stack} upgrade, you can decide whether to upgrade {agent}s to 8.0, which is recommended to ensure you get the latest features.
```